phpunit.xml

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="vendor/autoload.php">
+         bootstrap="bootstrap.php">
```

Comment:
Is using the vendor/autoload.php no longer required?

Reply:
We do use it, but its included in the bootstrap.php